### PR TITLE
Fix flaky split test

### PR DESCRIPTION
**Pull request by Konboi**
SSIA

### CODE REVIEW

File: .github/workflows/python-package.yml
Comment (Konboi) [inline .github/workflows/python-package.yml:14]: If I get approval, I'll delete this change before merging this PR

**Review by kohsuke — APPROVED** [review]
I don't know how you tracked this down, but bravo!

**Comment by kohsuke** [comment]
Oops, I incorrectly assumed this branch got merged to the main and messed things up.

**Pull request merged**
